### ISSUE TITLE
Can't find images for the music panel

### ISSUE THREAD

**Issue by DMyTryC** [schwabdidier/GazePlay#436]
When running from NetBeans IDE I receive the error :
```
2018-05-25 15:59:34,987 WARN  - java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found : data\common\images\previous.png
2018-05-25 15:59:35,098 WARN  - java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found : data\common\images\previous.png
2018-05-25 15:59:35,100 WARN  - java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found : data\common\images\pause.png
2018-05-25 15:59:35,102 WARN  - java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found : data\common\images\play-button.png
2018-05-25 15:59:35,104 WARN  - java.lang.IllegalArgumentException: Invalid URL: Invalid URL or resource not found : data\common\images\skip.png
```

Might be related to #359

**Comment by kuhlkrein** [comment]
Same Problem for me

**Comment by kuhlkrein** [comment]
Fixed by #487
